client/src/components/TeamMember/TeamMember.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from "react";
+import "./TeamMember.css";
+
+const TeamMember = props => {
```

Comment:
Would be great to get into the habit of destructuring here and in other components e.g. 

```
const TeamMember = {name} => {
...
<div>{name}</div>
```